proto/tenancy_config/rpc/kce_config_svc.proto

```diff
@@ -0,0 +1,57 @@
+syntax = "proto3";
+
+package holms.types.tenancy_config.rpc;
+option csharp_namespace = "HOLMS.Types.TenancyConfig.RPC";
+option go_package = "tenancyconfig/rpc";
+
+import "tenancy_config/kaba_config.proto";
+import "tenancy_config/miwa_config.proto";
+import "tenancy_config/active_kce_configuration.proto";
+import "google/protobuf/empty.proto";
+
+message GetKCEConfigResponse {
+	.holms.types.tenancy_config.KabaConfig kaba_configuration = 1;
+	.holms.types.tenancy_config.MiwaConfig miwa_configuration = 2;
+	.holms.types.tenancy_config.ActiveKCEConfiguration active_kce_config = 3;
+}
+
+message SetKabaConfigResponse {
+	KCESvcSetKabaConfigResult result = 1;
+}
+
+enum KCESvcSetKabaConfigResult {
+	SET_KCE_CONFIG_INVALID_IP_ADDRESS = 0;
+	SET_KCE_CONFIG_INVALID_ENCODER_NUMBER = 1;
+	SET_KCE_CONFIG_INVALID_PWD = 2;
+	SET_KCE_CONFIG_UNKNOWN_FAILURE = 3;
+	SET_KCE_CONFIG_SUCCESS = 4;
+}
+
+enum SetMiwaConfigResult {
+	SET_MIWA_CONFIG_INVALID_IP_ADDRESS = 0;
+	SET_MIWA_CONFIG_INVALID_TERMINAL_NUMBER = 1;
+	SET_MIWA_CONFIG_UNKNOWN_FAILURE = 2;
+	SET_MIWA_CONFIG_SUCCESS = 3;
+}
+
+message KCESvcSetMiwaConfigResponse {
+	SetMiwaConfigResult result = 1; 
+}
+
+message KCESvcSetActiveKCEConfigRequest {
+	.holms.types.tenancy_config.ActiveKCEConfiguration active_kce_config = 1;
+}
+
+service KCEConfigSvc {
+	rpc Get(google.protobuf.Empty) 
+		returns (GetKCEConfigResponse);
+
+	rpc SetKabaConfig(.holms.types.tenancy_config.KabaConfig)
```

Comment:
Can you comment a bit on the rationale for 3 rpc calls? It looks like the call pattern is "set active configuration", and then we store (potentially) both a MIWA and a KABA configuration separately?

What was the rationale for this design? Not knowing any better, I'd probably have designed it as a single configuration that could be switched between one or the other. Was the intention to have both in parallel to allow easy switching between the two? I think only one can be active at a time, right?